src/main/web_src/src/pages/minSide/maler/MalModal.js

```diff
@@ -0,0 +1,15 @@
+import DollyModal from '~/components/ui/modal/DollyModal'
+import { EndreMalnavn } from '~/pages/minSide/maler/EndreMalnavn'
+import React from 'react'
+
+export const MalModal = ({ bestilling, closeModal }) => (
+	<DollyModal isOpen closeModal={closeModal} width="60%">
```

Comment:
Skulle veldig gjerne hatt litt styling på denne modalen, for å få den til å se ut som resten av modalene våre 😄  Om det ikke blir for komplisert når du har gjenbrukt EndreMalnavn-komponenten 🤔 
Det jeg ønsker meg er:
- "Opprett ny mal" som h1
- "Navn på ny mal" som label på inputboksen.
- Bredere innputboks - kan gjerne fylle modalens bredde.
- Lagre-knapp som hoved-knapp.
- Bør også ha en avbryt-knapp til vestre for lagre-knappen.
- Modalen kan gjerne være litt smalere siden den ikke har så mye innhold.

Bare å si ifra om det er noe jeg kan hjelpe til med 😄

Reply:
Takk, veldig greit å noe konkret å jobbe mot 🤩  Supert det, skal si ifra hvis det blir vanskelig! 😄